src/pydocstyle.py

```diff
@@ -683,6 +683,10 @@ def to_rst(cls):
                                  'docstring text')
 D211 = D2xx.create_error('D211', 'No blank lines allowed before class '
                                  'docstring', 'found %s')
+D212 = D2xx.create_error('D212', 'Multi-line docstring summary does not start '
+                                 'on the first line.')
+D213 = D2xx.create_error('D213', 'Multi-line docstring summary does not start '
+                                 'on a separate line.')
```

Comment:
I prefer a positive phrasing for these - "Multi-line docstring summary should start on the first line" and "Multi-line docstring should start on a separate line".